docs/how-to-guides/api/use-the-legacy-api-via-http-requests.md

```diff
@@ -15,6 +15,34 @@ All methods take a list of mandatory arguments which you need to pass every time
 - `timestamp`: The time in UTC in ISO 8601 format, used to indicate the validity of the signature.
 - `version`: The version of the API, 2011-08-01 being the current one. It’s in the form of a date.
 
+## Authentication
```

Comment:
See: 
<img width="246" height="272" alt="image" src="https://github.com/user-attachments/assets/d28da20c-5305-4a03-8063-9ff3a51cfe8b" />

Are all of those H2 headers after **Authentication** relevant regardless of which auth method you choose? (creating the signature, passing values, passing files)